coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -6148,7 +6148,7 @@ def forward(self, input_data):
 
         inputs = [TensorType(name="input_data", shape=(1, 2, 3), dtype=np.int32)]
         self.run_compare_torch(
-            inputs, TestModel(), backend=backend, compute_unit=compute_unit
```

Comment:
The testing change in this file is awesome!

Reply:
Exposed some existing flakiness 😂